src/cmd-koji-upload

```diff
@@ -77,6 +77,8 @@ KOJI_CG_TYPES = {
 # might be needed.
 COMPRESSION_EXT = ["gz", "xz"]
 
+STATE_FILE_TPL = "/tmp/cosa-cmd-koji-upload-{name}-{version}-{release}"
```

Comment:
Maybe this should live within the `builds` directory?

Similar to https://github.com/coreos/coreos-assembler/blob/master/src/cmd-koji-upload#L108